js/core.js

```diff
@@ -2933,7 +2934,9 @@
                 rightString += separator;
 
             }
-            return "(" + leftString + ")=(" + rightString + ")";
+
+            var joinType = ((typeof isLeft === "boolean") ? (isLeft ? "left": "right") : "");
```

Comment:
If it is not a `left` join, it's a `right` join. But if the variable is not a boolean type, what type of join is it? Maybe add a comment about that here? I'm sure that's written in the `ermrest` spec though..

Reply:
Based on Karl's comment, maybe we want to use a left join if specified and don't specify anything else for the other cases.

Reply:
I have documented it in the jsdoc `false: right join, true: left join, other values: inner join`. Even though right now only left join and inner join are being used. Although I'm going to change it to `true: left join, other values: inner join`